config/samples/install_v2_0_0_rc3.yaml

```diff
@@ -0,0 +1,586 @@
+                    type: object
+                  type: array
+              type: object
+            serviceRef:
+              description: serviceRef is a reference to a Kubernetes Service and ServicePort.
+              properties:
+                name:
+                  description: Name is the name of the Service.
+                  type: string
+                port:
+                  anyOf:
+                    - type: integer
+                    - type: string
+                  description: Port is the port of the ServicePort.
+                  x-kubernetes-int-or-string: true
+              required:
+                - name
+                - port
+              type: object
+            targetGroupARN:
+              description: targetGroupARN is the Amazon Resource Name (ARN) for the
+                TargetGroup.
+              type: string
+            targetType:
+              description: targetType is the TargetType of TargetGroup. If unspecified,
+                it will be automatically inferred.
+              enum:
+                - instance
+                - ip
+              type: string
+          required:
+            - serviceRef
+            - targetGroupARN
+          type: object
+        status:
+          description: TargetGroupBindingStatus defines the observed state of TargetGroupBinding
+          properties:
+            observedGeneration:
+              description: The generation observed by the TargetGroupBinding controller.
+              format: int64
+              type: integer
+          type: object
+      type: object
+  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: false
+    - name: v1beta1
+      served: true
+      storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: kube-system/aws-load-balancer-serving-cert
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/name: aws-load-balancer-controller
+  name: aws-load-balancer-webhook
+webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: aws-load-balancer-webhook-service
+        namespace: kube-system
+        path: /mutate-v1-pod
+    failurePolicy: Fail
+    name: mpod.elbv2.k8s.aws
+    namespaceSelector:
+      matchExpressions:
+        - key: elbv2.k8s.aws/pod-readiness-gate-inject
+          operator: In
+          values:
+            - enabled
+    rules:
+      - apiGroups:
+          - ""
+        apiVersions:
+          - v1
+        operations:
+          - CREATE
+        resources:
+          - pods
+    sideEffects: None
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: aws-load-balancer-webhook-service
+        namespace: kube-system
+        path: /mutate-elbv2-k8s-aws-v1alpha1-targetgroupbinding
```

Comment:
do we plan to update the path as well?

Reply:
nice catch :D, yeah, the path should be updated. (and we should have a version in the path since different versions might be imcompatible)